routing/nexus_config.php

```diff
@@ -12,28 +12,17 @@ class Config
         public $dbName;
         public $dbUser;
         public $dbPass;
-        public function __construct($sitePath)
-        {
-            $db = new \PDO(
-                "mysql:host=".\Routing_Config::$dbHost.";dbname=".\Routing_Config::$dbName.";charset=UTF8",
-                \Routing_Config::$dbUser,
-                \Routing_Config::$dbPass,
-                array()
-            );
-            $sql = "SELECT * FROM orgchart_configs WHERE path = '$sitePath';";
-            $query = $db->prepare($sql);
-            $query->execute(array());
-            $res = $query->fetchAll(\PDO::FETCH_ASSOC);
-            
-            $this->title = $res[0]['title'];
-            $this->city = $res[0]['city'];
-            $this->adminLogonName = $res[0]['adminLogonName'];
-            $this->adPath = json_decode($res[0]['active_directory_path']);
-            $this->uploadDir = ltrim($res[0]['path'], "/") . $res[0]['upload_directory'];
-            $this->ocPath = ltrim($res[0]['path'], '/');
+        public function __construct($configData)
+        {            
+            $this->title = $configData[0]['title'];
+            $this->city = $configData[0]['city'];
+            $this->adminLogonName = $configData[0]['adminLogonName'];
+            $this->adPath = json_decode($configData[0]['active_directory_path']);
+            $this->uploadDir = ltrim($configData[0]['path'], "/") . $configData[0]['upload_directory'];
```

Comment:
Since we're storing the new UPLOADS location in leaf_config. Can we change this variable to be just ltrim($configData[0]['upload_directory'], '/')

Reply:
done